src/system/Nav/Nav.tsx

```diff
@@ -0,0 +1,72 @@
+/** @jsxImportSource theme-ui */
+import React, { Ref, forwardRef } from 'react';
+import * as NavigationMenu from '@radix-ui/react-navigation-menu';
+import classNames from 'classnames';
+
+import { VIP_NAV } from '.';
+import { ThemeUIStyleObject } from 'theme-ui';
+import { NavItemProps } from './NavItem';
+
+export interface NavProps extends NavigationMenu.NavigationMenuProps {
+	className?: string;
+	variant?: 'primary' | 'secondary' | 'display' | 'text' | 'tabs';
+	sx?: ThemeUIStyleObject;
+	label: string;
+}
+
+const Nav = forwardRef< HTMLElement, NavProps >(
+	(
+		{ className, children, orientation, variant = 'primary', sx = {}, label }: NavProps,
+		ref: Ref< HTMLElement >
+	) => {
+		const childrenWithVariant = React.Children.map(
+			children as React.ReactElement< NavItemProps >,
+			child => {
+				if ( React.isValidElement( child ) ) {
+					return React.cloneElement( child, { variant } );
+				}
+
+				return child;
+			}
+		);
```

Comment:
Whoa. I mean, i get it. And i'm no fan of render props. But `cloneElement` can cause problems. 

Can we just require the `variant` prop on `Nav.Item`? It's unlikely to be abused

Reply:
@chriszarate I think this is a bad experience. We would need to pass it to all `Nav.Item` this prop, which can lead to problems in the implementation, broken design, etc.

Reply:
I am open to suggestions, but here's the problem:

```jsx
<Nav.Root label="Etc">
  <Nav.Item variant="primary" href="#">PHP</Nav.Item>
  <Nav.Item variant="primary" href="#">PHP 2</Nav.Item>
</Nav.Root>
```

We need to repeat if we require it on the Item level.

Reply:
IMO repetition is better than `cloneElement` here, but I understand the reasoning. It's marked as legacy, so I think we should avoid it. It can also cause issues with TypeScript prop types.

https://react.dev/reference/react/legacy